atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -168,6 +168,10 @@ def get_srpm_urls(self, sigkeys=None, insecure=False):
         for rpm_id, rpm_build_id in rpm_build_ids.items():
             self.log.debug('Resolving SRPM for RPM ID: %s', rpm_id)
             rpm_hdr = self.session.getRPMHeaders(rpm_id, headers=['SOURCERPM'])
+            if 'SOURCERPM' not in rpm_hdr:
+                msg = ('RPM comes from an external repo (RPM ID: {}). '
```

Comment:
Is it possible that an RPM will lack the sourcerpm header for a different reason?
The entries from getRPMs contain a `external_repo_name` that do describe if the repository is `INTERNAL` or sth else.

Reply:
That seems more solid, thanks